src/Groups/libraries/atlasgroups.jl

```diff
@@ -0,0 +1,77 @@
+export
+    atlas_group,
+    number_atlas_groups
+
+###################################################################
+# Groups from the Atlas of Group Representations
+###################################################################
+
+"""
+    atlas_group([::Type{T}, ]name::String) where T <: Union{PermGroup, MatrixGroup}
+
+Return a group from the Atlas of Group Representations
+whose isomorphism type is given by `name` and have the type `T`.
+
+# Examples
+```jldoctest
+julia> atlas_group("A5")  # alternating group A5
+Group([ (1,2)(3,4), (1,3,5) ])
+
+julia> atlas_group(MatrixGroup, "A5")
+Matrix group of degree 4 over Galois field with characteristic 2
+
+julia> atlas_group("M11")  # Mathieu group M11
+Group([ (2,10)(4,11)(5,7)(8,9), (1,4,3,8)(2,5,6,9) ])
+
+```
+"""
+function atlas_group(name::String)
+  G = GAP.Globals.AtlasGroup(GAP.GapObj(name))
+  G === GAP.Globals.fail && error("the group atlas does not provide a representation for $name")
+  T = _get_type(G)
+  return T(G)
+end
+
+function atlas_group(::Type{T}, name::String) where T <: Union{PermGroup, MatrixGroup}
+  if T === PermGroup
+    G = GAP.Globals.AtlasGroup(GAP.GapObj(name), GAP.Globals.IsPermGroup, true)
+  else
+    G = GAP.Globals.AtlasGroup(GAP.GapObj(name), GAP.Globals.IsMatrixGroup, true)
+  end
+  G === GAP.Globals.fail && error("the group atlas does not provide a representation of type $T for $name")
+  TT = _get_type(G)
+  return TT(G)
```

Comment:
Shouldn't this still be:
```suggestion
  return T(G)
```

Reply:
This is delicate.
If we look at `_get_type` then we find that in the case of matrix groups, we do not get a type but a dedicated function that knows enough about the context in order to "do the right thing". I would be happy if we could simplify this.

Reply:
OK, let's leave it as it is for now then, we can worry about this later.